ssh_config.5

```diff
@@ -1486,6 +1486,12 @@ an OpenSSH Key Revocation List (KRL) as generated by
 .Xr ssh-keygen 1 .
 For more information on KRLs, see the KEY REVOCATION LISTS section in
 .Xr ssh-keygen 1 .
+.It Cm RSAMinimumModulusSize
+Specifies the minimum accepted RSA modulus size in different
+contexts.  This is only be used with older SSH servers when it is
```

Comment:
Line break after a full stop, so the spacing (american or proper) is applied automatically. See: https://man.openbsd.org/roff.7#Sentence_Spacing